sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/UnionEstimation.scala

```diff
@@ -103,10 +112,11 @@ object UnionEstimation {
                 }
                 (min, max)
             }
-          val newStat = ColumnStat(min = minMaxValue._1, max = minMaxValue._2)
+          val newStat = ColumnStat(min = minMaxValue._1, max = minMaxValue._2,
+            nullCount = nullCount)
           outputAttrStats += unionOutput(outputIndex) -> newStat
       }
-      AttributeMap(outputAttrStats.toSeq)
+      AttributeMap(outputAttrStats)
```

Comment:
`.toSeq` is needed for `newer` Scala compilers?

Reply:
Sure. Undid the change